metabrainz/templates/index/contact.html

```diff
@@ -52,36 +52,33 @@ <h1 class="page-title">Contact</h1>
       disallowed topics, we reserve the right to make fun of you.
     </em>
   </p>
+
+  <h3>E-mail us</h3>
   <button id="agree-btn" class="btn btn-default btn-success">
-    I agree that I will not contact you regarding the topics above
+    I agree with rules above, show me email address
```

Comment:
Is "I won't contact you about these issues, show me the email address" too long? At least please add the articles ("the rules" / "the email") :)